couchdb/server/mime.py

```diff
@@ -0,0 +1,211 @@
+# -*- coding: utf-8 -*-
+#
+import logging
+
+from pprint import pformat
+
+from couchdb.server.exceptions import Error
+from couchdb.util import OrderedDict
+
+log = logging.getLogger(__name__)
+
+__all__ = ['best_match', 'MimeProvider', 'DEFAULT_TYPES']
+
+
+def parse_mimetype(mimetype):
+    parts = mimetype.split(';')
+    params = {}
+    for item in parts[1:]:
+        if '=' in item:
+            key, value = item.split('=', 2)
```

Comment:
This code will lead to a [bug](https://github.com/KeepSafe/aiohttp/commit/9ad54c74fe8a8f58b89389ae66a809d17ea9ace2). Yes, I wrote it so, it works correct, but may cause an error (: